src/cascadia/TerminalApp/App.cpp

```diff
@@ -1257,8 +1257,9 @@ namespace winrt::TerminalApp::implementation
     {
         if (profile.HasIcon())
         {
-            auto path = profile.GetIconPath();
-            winrt::hstring iconPath{ path };
+            std::wstring path{ profile.GetIconPath() };
+            auto envExpandedPath{ wil::ExpandEnvironmentStringsW<std::wstring>(path.data()) };

```

Comment:
```suggestion
            const auto envExpandedPath{ wil::ExpandEnvironmentStringsW<std::wstring>(path.data()) };
```

Reply:
Is there a convention that local variables should be declared `const`? I can't find anything like that in the C++ Core Guidelines.

Reply:
We typically const the heck out of everything that can be consted. I don't know if it's a Core guideline, but it's something we have generally been following as a team.